YIF.Core.Data/Seaders/SeederDB.cs

```diff
@@ -2207,6 +2208,41 @@ public async static Task SeedInstitutionOfEducationModerators(EFDbContext contex
             }
         }
 
+        public async static Task SeedDepartments(EFDbContext context) 
+        {
+            if (context.Departments.Count() == 0)
+            {
+                await context.Departments.AddRangeAsync(new List<Department> 
+                {
+                    new Department 
+                    {
+                        Name = "Професійного та кар'єрного розвитку",
+                        Description = "Ваше майбутнє у великій мірі залежить від Вас особисто – від вашої наполегливості, вашого бажання зробити кар’єру."
+                    },              
+                    new Department
+                    { 
+                        Name = "Якості освіти", 
+                        Description = "Сприяє визнанню кваліфікацій, які здобувають студенти, та досвіду вищої освіти, який вони отримують, як головних пріоритетів університету" 
+                    },
+                    new Department 
+                    { 
+                        Name = "Зв’язків з громадкістю", 
+                        Description = "Систематичне та своєчасне, якісне і повне інформування про важливі події в університеті до міських, обласних, загальнодержавних засобів масової інформації" 
+                    },
+                    new Department 
+                    {
+                        Name = "Редакційно-видавничий", 
+                        Description = "Друк навчальних та навчально-методичних посібників, наукових видань та збірників наукових праць, методичних вказівок, довідкових та інформаційних матеріалів, дипломів та додатків до дипломів, авторефератів, бланкової продукції, журналів обліку та індивідуальних планів, студентських газет, запрошень, оголошень, грамот, вітальних адрес" 
+                    },
+                    new Department 
+                    { 
+                        Name = "Експлуатації комп'ютерних систем",
```

Comment:
change department names to make it look like "Професійний та кар'єрний розивоток", "Якість освіти" etc. instead of "Професійного та кар'єрного розвитку", "Якості освіти"

Reply:
Done